sdks/python/apache_beam/coders/coders.py

```diff
@@ -365,7 +365,7 @@ def maybe_dill_dumps(o):
   # We need to use the dill pickler for objects of certain custom classes,
   # including, for example, ones that contain lambdas.
   try:
-    return pickle.dumps(o)
```

Comment:
Use `https://docs.python.org/2/library/pickle.html#pickle.HIGHEST_PROTOCOL` to be more clear with the intent?